### PR TITLE
Simplified API

### LINKED ISSUE #74: Simplified API for basic usage
Add methods to `LTO` for creating, signing, and broadcasting transactions. Novice users shouldn't need to worry about such details.

```ts
const lto = new LTO('T');

lto.transfer(myAccount, recipient, 100 * 10^8);
lto.massTransfer(myAccount, [{recipient: '', amount: 100 * 10^8}, {recipient: '': amount: 50 * 10^8}]);
lto.anchor(myAccount, sha256('some value'), sha256('other value'));
lto.issueAssociation(myAccount, recipient, 10);
lto.revokeAssociation(myAccount, recipient, 10);
lto.lease(myAccount, recipient, 10000 * 10^8);
lto.cancelLease(myAccount, leaseId);
lto.sponsor(myAccount, otherAccount);
lto.cancelSponsorship(myAccount, otherAccount);

lto.getBalance(myAccount);
lto.setData(myAccount, {foo: 'bar'});
lto.getData(myAccount);
```

For more advanced features like sponsoring transactions, you'll still need to create a `Transaction` object.

### PR DESCRIPTION
**Pull request by jasny**
Closes #74

### CODE REVIEW

File: src/LTO.ts [hunk 1]
```diff
@@ -56,62 +67,120 @@ export default class LTO {
...
-		if (encryptionRounds < 1000)
-			console.warn("Encryption rounds may be too few");
 
-		if (seedPhrase.length < DEFAULT_CONFIG.minimumSeedLength)
-			throw new Error("The seed phrase you are trying to encrypt is too short");
+	/**
+	 * Transfer LTO from account to recipient.
+	 * Amount is number of LTO * 10^8.
+	 */
+	public transfer(sender: Account, recipient: string, amount: number, attachment: Uint8Array|string = ""): Promise<Transfer> {
```
Comment (brunodb3) [inline src/LTO.ts:102]: For the transactions, what if we want to sponsor it before broadcasting? How would we use it?

File: test/accounts/ed25519.spec.ts [hunk 2]
```diff
@@ -69,17 +69,4 @@ describe('ed25519 account', () => {
       expect(ec.id).to.eq('2bGCW3XbfLmSRhotYzcUgqiomiiFL7y3h7xPH8Ha3FVaYFp6YsjzSRSBeVjnae');
     });
   });
-
-  describe('#encryptSeed', () => {
-    it('should generate a cypher text from seed phrase', () => {
```
Comment (brunodb3) [inline test/accounts/ed25519.spec.ts:74]: Is this test not needed anymore?

**Review by brunodb3 — CHANGES_REQUESTED** [review]
Could you add the changes to the `README.md`? Seems that the way to use the lib has changed quite a bit with a simplified version, that should be reflected on the `README`

File: test/accounts/ed25519.spec.ts (hunk 2 above)
Comment (jasny) [reply test/accounts/ed25519.spec.ts:74]: The code was not in the correct place. Plus the test was not good. We need to write more tests and will add this.

**Comment by jasny** [comment]
> Could you add the changes to the `README.md`? Seems that the way to use the lib has changed quite a bit with a simplified version, that should be reflected on the `README`

The README and documentation need to be updated, but that not part of this PR.

**Pull request merged**